apps/token-manager/test/tokenmanager.js

```diff
@@ -1,465 +1,440 @@
-const { assertRevert } = require('@aragon/test-helpers/assertThrow')
-const { getEventArgument, getNewProxyAddress } = require('@aragon/test-helpers/events')
-const getBalance = require('@aragon/test-helpers/balance')(web3)
-const { makeErrorMappingProxy } = require('@aragon/test-helpers/utils')
+const ERRORS = require('./helpers/errors')
+const { assertBn, assertRevert } = require('@aragon/contract-helpers-test/src/asserts')
+const { ANY_ENTITY, getInstalledApp, encodeCallScript } = require('@aragon/contract-helpers-test/src/aragon-os')
+const { getEventArgument, injectWeb3, injectArtifacts, ZERO_ADDRESS } = require('@aragon/contract-helpers-test')
 
-const { encodeCallScript } = require('@aragon/test-helpers/evmScript')
-const ExecutionTarget = artifacts.require('ExecutionTarget')
+injectWeb3(web3)
+injectArtifacts(artifacts)
```

Comment:
I think we need the artifacts at least in all of them to use `getInstalledApp` since it requires the Kernel ABI. But I agree with you with being explicit

Reply:
The artifacts should also be implicitly injected into the test helpers, similar to web3.